spec/requests/lanes_spec.rb

```diff
@@ -20,7 +20,7 @@
                       status: 'completed',
                       active_version: true)
     FactoryBot.create(:workflow_step,
-                      process: 'accept',
+                      process: 'shelve-complete',
```

Comment:
these changes to process names (here and all the rest below) are necessary because this new validation requires valid step names, even in tests